README.md

```diff
@@ -10,15 +10,17 @@ You are only permitted to have *one* Commander. Commanders with Partner may be u
 
 ## Banlist
 
-_Correct as of 2019-03-13_ 
+_Correct as of 2019-03-14_ 
 
 - [Standard Commander ban list](https://scryfall.com/search?q=banned%3Aedh&order=name&as=grid&unique=cards)
+- Commanders with Keyword: "Partner" (You may play one of the partner commanders, but you may not have both in the command zone)
 
 plus
 
 - [Cranial Plating](https://scryfall.com/card/mm2/206/cranial-plating)
 - [Demonic Tutor](https://scryfall.com/card/lea/104/demonic-tutor)
 - [Empty the Warrens](https://scryfall.com/card/dds/15/empty-the-warrens)
 - [Frantic Search](https://scryfall.com/card/uma/57/frantic-search)
+- [Library of Alexandria](https://scryfall.com/card/vma/303/library-of-alexandria)
```

Comment:
Library of Alexandria is already in the standard Commander ban list and as such does not need explicitly calling out.

Reply:
Fair shout, Didn't see it on my prior perusal.